addons/mail/static/src/components/discuss_sidebar/discuss_sidebar.scss

```diff
@@ -1,42 +1,11 @@
-// ------------------------------------------------------------------
-// Layout
-// ------------------------------------------------------------------
-
 .o_DiscussSidebar {
-    display: flex;
-    flex-flow: column;
     width: $o-mail-chat-sidebar-width;
 
     @include media-breakpoint-up(xl) {
         width: $o-mail-chat-sidebar-width + 50px;
     }
 }
 
-.o_DiscussSidebar_category {
-    display: flex;
-    flex-flow: column;
-    flex: 0 0 auto;
-}
-
 .o_DiscussSidebar_quickSearch {
-    border-radius: 10px;
-    margin: 0 $o-mail-discuss-sidebar-scrollbar-width 10px;
-    padding: 3px 10px;
-}
```

Comment:
This negative diff on `o_DiscussSidebar_quickSearch` is on the wrong commit.
- Currently in `[REF] mail: discuss_sidebar scss refactoring`
- Should be in `[IMP] mail: discuss_sidebar design revamp`

Reply:
Done! 👍